litmus-portal/README.md

```diff
@@ -88,28 +88,7 @@ View the User Guide <b>[here](https://docs.litmuschaos.io/)</b>
 
 ### **Uninstallation**
 
-> Litmus-2.0.0 (Stable)
-```bash
-kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/2.0.0/docs/2.0.0/litmus-2.0.0.yaml
-```
-
-Or
-
-> Master (Latest) Cluster scope. Uninstall in litmus namespace by default. (Only deleting litmus portal components)
-```bash
-kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/cluster-k8s-manifest.yml
-```
-
-Or
-
-> Master (Latest) Namespaced scope. Replace `<namespace>` with the desired namespace.
-```bash
-export LITMUS_PORTAL_NAMESPACE="<namespace>"
-kubectl delete -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/litmus-portal-crds.yml
-kubectl delete -f ${LITMUS_PORTAL_NAMESPACE}-ns-scoped-litmus-portal-manifest.yml -n ${LITMUS_PORTAL_NAMESPACE}
-```
-
-##### **Additional information**
+You can uninstall the steps using from [section here](https://litmusdocs-beta.netlify.app/docs/user-guides/uninstall-litmus)
```

Comment:
should we use docs.litmuschaos.io link instead of beta

Reply:
I Will update in the next PR.. Merged it.